numpy/_core/tests/test_multiarray.py

```diff
@@ -10301,6 +10301,12 @@ def test_argsort_int(N, dtype):
     arr[N - 1] = maxv
     assert_arg_sorted(arr, np.argsort(arr, kind='quick'))
 
+# Test large arrays that leverage openMP implementations from x86-simd-sort:
+@pytest.mark.parametrize("dtype", [np.float16, np.float32, np.float64])
+def test_sort_largearrays(dtype):
+    N = 1000000
+    arr = np.random.rand(N)
```

Comment:
maybe we should pin the values down with modern `default_rng`?

One other thing I checked was how slow the test might be (do we want a `slow` marker for the large array handling?). It didn't seem too bad (< 1 s for each case locally on an ARM Mac), though it is in the top 10 slowest for this module for example.

Reply:
I did pin down the value with a determined seed.